torchvision/models/detection/retinanet.py

```diff
@@ -691,7 +696,17 @@ class RetinaNet_ResNet50_FPN_Weights(WeightsEnum):
 
 
 class RetinaNet_ResNet50_FPN_V2_Weights(WeightsEnum):
-    pass
+    COCO_V1 = Weights(
+        url="",
+        transforms=ObjectDetection,
+        meta={
+            **_COMMON_META,
+            "num_params": 38198935,
```

Comment:
Question: Since the architecture comes from paper `Bridging the Gap Between Anchor-based and Anchor-free Detection via Adaptive Training Sample Selection` that published in 2019, shouldn't we have `publication_year: 2019` here?

Reply:
This is a good point. The specific paper doesn't really describe the retinanet architecture but rather proposes a few optimizations to close the gap between anchor-based and anchor-free architectures. So a very good question is, what the `publication_year` should contain? Somehow I felt that it should still refer to the original paper but no strong opinion, happy to change it.

Another possibility is to remove completely the `publication_year` from all meta-data. Some background info on why it's there: I've added it few months back when I was doing some analysis on the volumes of the model weights downloads from the S3 bucket to understand which models we should focus on improving. Back then on my stats script I was also checking how old the model is hence why this exists in here. But perhaps it creates more issues than is fixes; the paper is missing after all. Thoughts?

Reply:
I'm going to update the publication_year here to unblock but let's create an issue to remove the `publication_year` field from all meta-data. I'm OK to review other ways to encode this, but this single numeric field is not the right way to do it. What happens if there are multiple papers?